dbt-cta/quickbooks/models/1_cta_full_refresh/_1_cta_full_refresh__models.yml

```diff
@@ -0,0 +1,6918 @@
+version: 2
+
+models:
+
+  - name: tax_agencies_MetaData_base
+    description: ''
+    columns:
+      - name: _airbyte_tax_agencies_hashid
+        description: ''
+      - name: createtime
+        description: ''
+      - name: lastupdatedtime
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+      - name: _airbyte_emitted_at
+        description: ''
+      - name: _airbyte_normalized_at
+        description: ''
+      - name: _airbyte_MetaData_hashid
+        description: ''
+        tests:
+          - unique
+
+  - name: items_IncomeAccountRef_base
+    description: ''
+    columns:
+      - name: _airbyte_items_hashid
+        description: ''
```

Comment:
Think this is also 1-1 items to income account ref. [docs](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/item)